simulation/traffic_simulator/src/entity/entity_manager.cpp

```diff
@@ -38,12 +39,27 @@ namespace entity
 void EntityManager::broadcastEntityTransform()
 {
   std::vector<std::string> names = getEntityNames();
-  for (const auto & name : names) {
-    geometry_msgs::msg::PoseStamped pose;
-    pose.pose = getMapPose(name);
-    pose.header.stamp = clock_ptr_->now();
-    pose.header.frame_id = name;
-    broadcastTransform(pose);
+  if (entityExists("ego")) {
```

Comment:
Please do not use "ego" directly and use `getEgoName` instead.

Reply:
But this is just a nit, and does not block merging.

Reply:
Fixed by https://github.com/tier4/scenario_simulator_v2/pull/1238/commits/61af6463271fb24640a32fbb075e921d31677cb0